sklearn/ensemble/tests/test_iforest.py

```diff
@@ -44,7 +44,7 @@
 diabetes.target = diabetes.target[perm]
 
```

Comment:
If you could still remove line 30 and locally create a random state in the function that requires one (or just pass an integer) it would be better.

Reply:
Updated as suggested :)